src/OneExplorer.ts

```diff
@@ -177,3 +183,35 @@ export class OneExplorer {
         vscode.window.registerTreeDataProvider('OneExplorerView', oneTreeDataProvider));
   }
 }
+
+//
+// menu handler
+//
+
```

Comment:
IMO, I doubt if this is right to be included in OneExplorer.ts. 

I thought OneExplorer will only provide a UI and this runner functionality will be performed by another module.

Reply:
I have the same doubt, too. 😸 
I wanted to put this code as a separate file but in the same directory of `OneExplorer.ts` since this code is implementation of menu of `OneExplorer`.

However, `OneExplorer.ts` is in `src` and I didn't want to create more file inside `src.

How about the following later?
- move `OneExplorer.ts` into some dir for code related to OneExplorer
  - note: @jyoungyun is writing previous Compile View under `Compile` directory.
  - then we can extract this code into a a file inside the dir fo `OneExplorer.ts`.

If you agree, I will add some `TODO` inside this file.

Reply:
Now I understand why you have suggested making a directory of OneExplorer. I agree. 😄